docs/creating_integration_manifest.md

```diff
@@ -26,6 +26,7 @@ Or a minimal example that you can copy into your project:
 {
   "domain": "your_domain_name",
   "name": "Your Integration",
+  "version": "0.01",
```

Comment:
It's only required for custom integrations, not core integrations.

Reply:
That's what I was doing.  I was trying to create a custom integration from the Create your First Integration page.

Reply:
As the version is only required for custom integrations the minimal example is correct.

Reply:
I'll close this.  Probably two examples are required for minimal in the two folders.